eks-s3-seldon-mlflow/README.md

```diff
@@ -1,8 +1,19 @@
-# EKS, S3, Seldon and MLflow Stack 
+# 🥗 EKS, S3, RDS, MLflow and Seldon MLOps Stack Recipe 
 
-This is the stack recipe for having EKS as the orchestrator, S3 as the artifact store, Seldon Core as the model deployer and MLflow as the experiment tracker.
+Once you have gotten a hang of what ZenML is and how basic pipelines work, it is only natural to have the thought of deploying the pipeline to a cloud environment. There can be many motivations behind this, from neeeding specialized compute 💪 for training jobs to having a 24x7 load-balanced deployment of your trained model serving user requests 🚀.
```

Comment:
I think we can be a bit less ZenML specific here right? I mean the repo is meant also as a general repository to deploy these tools. Otherwise I like the intro you make an the paragraph below.

Perhaps start in a generic way and then mention ZenML at the end. "This stack can be used alongside the ZenML MLOps framework"

Reply:
Agreed. I think we can perhaps keep the cloud guide documentation page reference in the next paragraph because most of it is framework agnostic too?

I was also thinking of adding links to our (future) documentation pages on orchestrator, etc. while defining the stack. For example,
> - An EKS cluster that can act as an [orchestrator](docs) for your pipelines. 
> - An S3 ...

 This is coming from the idea that the new doc page would be focused on a general public too and designed as a source of learning about MLOps stacks. Would be cool imo.